### PR TITLE
required metadata stuff 

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

File: code.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/WMA-Data-Science/dataRetrieval/blob/master/LICENSE.note"
+        }
+        ]
+    },
+    
+    "homepageURL": "https://github.com/USGS-R/geoknife",
```
Comment (ldecicco-USGS) [inline code.json]: You'll want to change this to the code.usgs.gov address. That's the "OFFICIAL HOME"

File: DESCRIPTION [hunk 2]
```diff
@@ -53,4 +53,4 @@ Suggests:
     rgdal,
```
Comment (ldecicco-USGS) [inline DESCRIPTION]: Consider adding those ORCIDs?
https://github.com/USGS-R/EGRET/blob/master/DESCRIPTION#L5

**Comment by wdwatkins** [comment]
Ok.  I was going to wait to get these merged before I pushed it there.  I guess I should just do that now though

File: cran-comments.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ## Test environments
 * local OS X install, R 3.5.1
 * travis (Ubuntu 14.04.5 LTS, R 3.5.0)
-* win-builder
+* win-builder (R-devel)
```
Comment (ldecicco-USGS) [inline cran-comments.md]: :100:

**Comment by jread-usgs** [comment]
Looks good, is the new roxygen the reason these `man/abstract-datagroup.Rd` files get dropped?

**Comment by wdwatkins** [comment]
The new `roxygen` dropped a lot of duplicate lines like these in the Rds:
image
I don't see any whole files that are dropped though?

**Comment by jread-usgs** [comment]
like these files?
image

**Comment by jread-usgs** [comment]
Gotcha, looks good.

**Pull request merged**